Resources/Prototypes/Entities/Clothing/Back/duffel.yml

```
@@ -230,3 +230,5 @@
       shader: unshaded
   - type: Storage
     capacity: 9999
+ - type: ClothingSpeedModifier
+    sprintModifier: 1 // makes its stats identical to other variants of bag of holding
```

indentation is incorrect

```suggestion
  - type: ClothingSpeedModifier
    sprintModifier: 1 // makes its stats identical to other variants of bag of holding

```